js/ui/grid_core/ui.grid_core.columns_controller.js

```diff
@@ -331,29 +331,112 @@ module.exports = {
+            * @type_function_param2_field9 column:dxTreeListColumn
+            * @type_function_param2_field10 rowType:string
+            * @type_function_param2_field11 watch:function
+            */
+
+            /**
+             * @name dxDataGridColumn.headerCellTemplate
+             * @type template|function
+             * @type_function_param1 columnHeader:dxElement
+             * @type_function_param2 headerInfo:object
+             * @type_function_param2_field1 component:dxDataGrid
+             * @type_function_param2_field2 columnIndex:number
+             * @type_function_param2_field3 column:dxDataGridColumn
              */
+
             /**
-             * @name GridBaseColumn.headerCellTemplate
+             * @name dxTreeListColumn.headerCellTemplate
              * @type template|function
              * @type_function_param1 columnHeader:dxElement
              * @type_function_param2 headerInfo:object
+             * @type_function_param2_field1 component:dxTreeList
+             * @type_function_param2_field2 columnIndex:number
+             * @type_function_param2_field3 column:dxTreeListColumn
+             */
+
+            /**
+             * @name dxDataGridColumn.editCellTemplate
+             * @type template|function
+             * @type_function_param1 cellElement:dxElement
+             * @type_function_param2 cellInfo:object
+             * @type_function_param2_field1 setValue(newValue, newText):any
+             * @type_function_param2_field2 data:object
+             * @type_function_param2_field3 component:dxDataGrid
+             * @type_function_param2_field4 value:any
+             * @type_function_param2_field5 displayValue:any
+             * @type_function_param2_field6 text:string
+             * @type_function_param2_field7 columnIndex:number
+             * @type_function_param2_field8 rowIndex:number
+             * @type_function_param2_field9 column:dxDataGridColumn
+             * @type_function_param2_field10 rowType:string
              */
+
             /**
-             * @name GridBaseColumn.editCellTemplate
+             * @name dxTreeListColumn.editCellTemplate
              * @type template|function
              * @type_function_param1 cellElement:dxElement
              * @type_function_param2 cellInfo:object
+             * @type_function_param2_field1 setValue(newValue, newText):any
+             * @type_function_param2_field2 data:object
+             * @type_function_param2_field3 component:dxTreeList
+             * @type_function_param2_field4 value:any
+             * @type_function_param2_field5 displayValue:any
+             * @type_function_param2_field6 text:string
+             * @type_function_param2_field7 columnIndex:number
+             * @type_function_param2_field8 rowIndex:number
+             * @type_function_param2_field9 column:dxTreeListColumn
+             * @type_function_param2_field10 rowType:string
```

Comment:
Add doc comment for row field:  "row:dxTreeListRow"
Add doc comment for watch field